doc/tutorials/content/gpu_install.rst

```diff
@@ -2,85 +2,44 @@
-us know on the user mailing list if you have tested this on other
-distributions.
+In this tutorial you will learn how to make your configure your system to make it compatible to run the GPU methods provided by PCL.
+This tutorial is for Ubuntu, other Linux distrubutions can follow a similar process to set it up.  
 
-The explanation
----------------
-
-In order to run the code you'll need a decent Nvidia GPU with Fermi or Kepler architecture you can check this by::
-
- $ lspci | grep nVidia
-
-This should indicate which GPU you have on your system, if you don't have an Nvidia GPU, we're sorry, but you
-won't be able to use PCL GPU.
-The output of this you can compare with `this link <http://www.nvidia.co.uk/object/cuda-parallel-computing-uk.html>`_  
-on the Nvidia website, your card should mention compute capability of 2.x (Fermi) or 3.x (Kepler) or higher.
-If you want to run with a GUI, you can also run::
-
- $ nvidia-settings
-
-From either CLI or from your system settings menu. This should mention the same information.
-
-First you need to test if your CPU is 32 or 64 bit, if it is 64-bit, it is preferred to work in this mode.
-For this you can run::
-
-  $ lscpu | grep op-mode
-
-As a next step you will need a up to date version of the Cuda Toolkit. You can get this 
-`here <http://developer.nvidia.com/cuda/cuda-downloads>`_, at the time of writing the
-latest version was 4.2 and the beta release of version 5 was available as well.
-
-First you will need to install the latest video drivers, download the correct one from the site
-and run the install file, after this, download the toolkit and install it.
-At the moment of writing this was version 295.41, please choose the most up to date one::
-
-  $ wget http://developer.download.nvidia.com/compute/cuda/4_2/rel/drivers/devdriver_4.2_linux_64_295.41.run
-
-Make the driver executable::
-
- $ chmod +x devdriver_4.2_linux_64_295.41.run
-
-Run the driver::
+Windows is nto currently officialy supported for the GPU methods.
 
- $ sudo ./devdriver_4.2_linux_64_295.41.run
-
-You need to run this script without your X-server running. You can shut your X-server down as follows:
-Go to a terminal by pressing Ctrl-Alt-F1 and typing::
+The Explanation
+---------------
 
- $ sudo service gdm stop
+In order to run the code you will need a system with an Nvidia GPU, having CUDA Toolkit v9.2+ installed. 
+We will not be covering CUDA toolkit installation in this tutorial as there alredy exists many great tutorials for the same.
+You can check your CUDA toolkit version using the following command:
+ $ nvcc --version | grep "release" | awk '{print $6}' | cut -c2-
 
-Once you have installed you GPU device driver you will also need to install the CUDA Toolkit::
+The GPU methods in PCL only work with GCC 7 onwards. This will not be a problem if you are running Ubuntu 18+, in case you are running Ubuntu 16, you will need to install GCC 7 as the default one available is GCC 5.
```

Comment:
Any limit on clang?

Reply:
Not aware as far as I remember. If I come across it will update the guide.